excellent/functions/builtin.go

```diff
@@ -1160,16 +1160,17 @@ func RandBetween(env envs.Environment, min types.XNumber, max types.XNumber) typ
 // * `YY`        - last two digits of year 0-99
 // * `YYYY`      - four digits of year 0000-9999
 // * `M`         - month 1-12
-// * `MM`        - month 01-12
+// * `MM`        - month, zero padded 01-12
 // * `D`         - day of month, 1-31
-// * `DD`        - day of month, zero padded 0-31
+// * `DD`        - day of month, zero padded 01-31
```

Comment:
something to note is a layout element like `DD` (`02` in go's format) doesn't parse non-zero padded inputs, but `D` (`2` in go's format) will parse zero-padded. 

So users are generally better off trying to parse with layouts like `D/M/YYYY` etc